test/DeltaTriplesTest.cpp

```diff
@@ -6,50 +6,17 @@
 
 #include <gtest/gtest.h>
 
-#include "./util/GTestHelpers.h"
-#include "./util/IndexTestHelpers.h"
+#include "DeltaTriplesTestHelpers.h"
```

Comment:
```suggestion
#include "./DeltaTriplesTestHelpers.h"
```

Reply:
Why do we want to include this header by it's relative path? Is it because this header only "belongs" to that single test?